controller/node_controller.go

```diff
@@ -409,6 +409,8 @@ func (nc *NodeController) syncNode(key string) (err error) {
 		condition.Reason = string(types.NodeConditionReasonManagerPodMissing)
 		condition.Message = fmt.Sprintf("manager pod missing: node %v has no manager pod running on it", node.Name)
 		node.Status.Conditions[types.NodeConditionTypeReady] = condition
+		// set node unschedulable
+		node.Spec.AllowScheduling = false
```

Comment:
Don't quite understand why this allows the node to be deleted... User should still able to disable scheduling on the node manually.

Reply:
You're right, this condition alone doesn't let the node to be deleted. The below changes are required as well.

Reply:
Feel like we don't need this. User should able to set it manually in case the node is permanently gone (need to double confirm).

Reply:
Apparently, there is no option for the user to disable scheduling in UI.